### DISCUSSION TITLE
Error: Could not determine if URL for media download is publicly routable

### DISCUSSION THREAD

**Discussion by abhinavmathur** [deepgram/community#665] [General help] [answered]
### Which Deepgram product feature are you using?

Deepgram API - STT Pre-recorded

### Details

Hello,

We are getting the following error:-
{"err_code":"REMOTE_CONTENT_ERROR","err_msg":"Could not determine if URL for media download is publicly routable.","request_id":"524919f1-d510-42c3-a6b5-e716b4d8418c"}

This error started happening today (March 27th) and is affecting all the transcriptions happening on our podcast hosting site. We checked the location of the file based on the error and it is publicly available. People are currently listening to the audio on distribution platforms like Apple and Spotify. This is the publicly available file sitting on our CDN:-
https://audio-delivery.cohostpodcasting.com/audio/7d408271-1f41-4337-9cdd-ac45711d73e7/episodes/4080c73a-93f5-49bf-8eba-fb4d98fd89c4/episode.mp3

### If you are making a request to the Deepgram API, what is the full Deepgram URL you are making a request to?

https://api.deepgram.com/v1/listen?punctuate=true&diarize=true&summarize=true&detect_topics=true&tier=nova

### If you are making a request to the Deepgram API and have a request ID, please paste it below:

524919f1-d510-42c3-a6b5-e716b4d8418c

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by nickottocode** [comment]
Hi @abhinavmathur 
I'm not sure what might have caused the original error message "Could not determine if URL for media download is publicly routable.","request_id" but it does appear to be resolving normally now.  Do you know if any recent changes were made to the CDN or DNS settings?

**Comment by aaronstarky** [comment]
Did anybody get an answer to this? I am encountering the same issue.

**Reply by jpvajda** [reply]
@aaronstarky  can you provide us a request ID so we can take a deeper look?

**Reply by aaronstarky** [reply]
Here is the request id: ef2a62e5-4a4e-456e-8110-7a160daf20f9

**Reply by jpvajda** [reply]
@aaronstarky  All I see is status unknown in our logs. 🤔 

```Status Code: Unknown. The request may still be processing or the response may not have been recorded in the Console DB. Check the infrastructure logs below for details.```

Can you share the API request you made with the parameters and also a sample of the audio file you were trying to send?

**Reply by aaronstarky** [reply, marked as answer]
I was able to figure out a workaround. Thank you for being willing to help :)

**Reply by jpvajda** [reply]
@aaronstarky  That's good news! What was the work around?

**Comment by shahbaz-khan-hub** [comment]
I am facing the same issue, but my URL is accessible to the browser. Can you please share how you resolved it?

**Comment by drsh4dow** [comment]
I'm facing the same issue, what was the workaround @aaronstarky ?

**Comment by victor-torres** [comment]
Hey, I've just got a failed request here for the same reason. 

```
Could not transcribe audio: [400] {"err_code":"REMOTE_CONTENT_ERROR","err_msg":"Could not determine if URL for media download is publicly routable.","request_id":"d2dd2a95-5a1d-4d22-bcbc-30f2917f4c91"}.
```

It was a temporary Amazon S3 link that should have been available / not expired during the time of the request. 
Unfortunately, I can't share the audio file or the original URL here. 

- @jpvajda, could you help me with this request or possibly explain how you try to determine if media downloads are publicly routable? 
- @aaronstarky, could you share what workaround are you using to fix this issue on your side?

**Reply by aaronstarky** [reply]
Hey! I can get you that workaround once I get to my codebase, its been a while since I worked on this project.

**Reply by AbbasPlusPlus** [reply]
Hey @aaronstarky can you please share it. I have also faced the exact same issue as @victor-torres
